src/Kernel32.Tests.Shared/UseCultureAttribute.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Threading;
+using Xunit.Sdk;
+
+/// <summary>
+/// Apply this attribute to your test method to replace the
+/// Thread.CurrentThread <see cref="CultureInfo.CurrentCulture" /> and
+/// <see cref="CultureInfo.CurrentUICulture" /> with another culture.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+internal class UseCultureAttribute : BeforeAfterTestAttribute
```

Comment:
Please, take a look on the new push. Now it makes sense and I got it to work without the `Thread` class (which is not present on the Profile 111)